ocpairprogramming/src/main/java/com/pino/project/ocpairprogramming/java8/ocp/chapter3/generics/Crate.java

```diff
@@ -0,0 +1,11 @@
+package com.pino.project.ocpairprogramming.java8.ocp.chapter3.generics;
+
+public class Crate<T> {
+	private T/*Object*/ contents;
+	//You can't create static variable as generic because type is linked to instance
+	//private static T property;//COMPILER ERROR
```

Comment:
we can add here other things that cannot be done with generic type

Reply:
The fact of the matter is that ma aim was to reproduce the "Type Erasure" applied by the compiler in order to be compatible with Raw Collection (Before java 5)